seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/common/SeaTunnelRowAccessor.java

```diff
@@ -45,4 +45,8 @@ public Object getField(int pos) {
     public Object[] getFields() {
         return row.getFields();
     }
+
+    public SeaTunnelRow getRow() {
```

Comment:
If we have `getRow` method, the whole `SeaTunnelRowAccessor` looks unnecessary. Please add new method with fields you want to get. Do not add method get row itself.